crates/sui-framework/build.rs

```diff
@@ -167,20 +170,28 @@ fn build_packages_with_move_config(
     serialize_modules_to_file(move_stdlib, &out_dir.join(stdlib_dir)).unwrap();
     // write out generated docs
     // TODO: remove docs of deleted files
-    for (fname, doc) in deepbook_pkg.package.compiled_docs.unwrap() {
-        let mut dst_path = PathBuf::from(DOCS_DIR);
-        dst_path.push(fname);
-        fs::write(dst_path, doc).unwrap();
-    }
-    for (fname, doc) in system_pkg.package.compiled_docs.unwrap() {
-        let mut dst_path = PathBuf::from(DOCS_DIR);
-        dst_path.push(fname);
-        fs::write(dst_path, doc).unwrap();
-    }
-    for (fname, doc) in framework_pkg.package.compiled_docs.unwrap() {
-        let mut dst_path = PathBuf::from(DOCS_DIR);
-        dst_path.push(fname);
-        fs::write(dst_path, doc).unwrap();
+    if write_docs {
```

Comment:
Generate docs is already a flag in the BuildConfig? What have a separate flag here?

Reply:
@tzakian could you leave a small comment here?